terraform/aks/workspace-variables/csv-sandbox.tfvars.json

```diff
@@ -0,0 +1,47 @@
+{
+  "env_config": "csv-sandbox",
+  "app_environment": "csv-sandbox",
+  "key_vault_name": "s189p01-rtt-csv-kv",
+  "key_vault_app_secret_name": "REGISTER-APP-SECRETS-CSV-SANDBOX",
+  "key_vault_infra_secret_name": "BAT-INFRA-SECRETS-PRODUCTION",
+  "cluster": "production",
+  "namespace": "bat-production",
+  "db_sslmode": "prefer",
+  "main_app": {
+    "main": {
+      "startup_command": [
+        "/bin/sh",
+        "-c",
+        "STATEMENT_TIMEOUT=90s bundle exec rails db:migrate && unset STATEMENT_TIMEOUT && bundle exec rails server -b 0.0.0.0"
+      ],
+      "probe_path": "/ping",
+      "replicas": 2,
+      "memory_max": "1536Mi"
+    }
+  },
+  "worker_apps": {
+    "worker": {
+      "startup_command": [
+        "/bin/sh",
+        "-c",
+        "bundle exec sidekiq -C config/sidekiq.yml"
+      ],
+      "replicas": 0,
+      "memory_max": "1Gi"
+    }
+  },
+  "azure_maintenance_window": {
+    "day_of_week": 0,
+    "start_hour": 2,
+    "start_minute": 0
+  },
+  "postgres_flexible_server_sku": "GP_Standard_D2ds_v4",
+  "azure_enable_backup_storage": false,
```

Comment:
set azure_enable_backup_storage to true as will make it easier to restore data into the new db